internal/agent/device/resource/memory.go

```diff
@@ -134,12 +82,6 @@ func (m *MemoryMonitor) Alerts() []v1alpha1.ResourceAlertRule {
 	return firing
 }
 
-func (m *MemoryMonitor) Usage() *MemoryUsage {
```

Comment:
usage was removed because we no longer persist the previous usage all in memory state is tracked in the alert.